content/master/concepts/pods.md

```diff
@@ -30,7 +30,7 @@ defining webhook configurations.
 The core CRDs installed by the init container include: 
 * CompositeResourceDefinitions, Compositions, Configurations and Providers
 * Locks to manage package dependencies
-* ControllerConfigs to apply settings to installed Providers
+* DeploymentRuntimeConfigs to apply settings to installed Providers
```

Comment:
```suggestion
* DeploymentRuntimeConfigs to apply settings to installed Providers and Functions
```

Perhaps? I'm okay leaving it off too, since they're beta.